packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2314,6 +2314,31 @@ export class ContainerRuntime
 		return result;
 	}
 
+	/**
+	 * Returns an aliased data store with the given alias.
+	 * @param alias - The alias for the data store.
+	 * @returns - The aliased data store as a FluidObject. Throws usage error if the data store
+	 * is not aliased or if it doesn't exist.
+	 */
+	public async getAliasDataStore(alias: string): Promise<FluidObject> {
```

Comment:
Should it be `getAliasedDataStore`?

Reply:
That's what I started with. But `getAliasDataStore` sounded better to me. Similar to `getRootDataStore` vs `getRootedDataStore`.
I am fine with `getAliasedDataStore` as well.

Reply:
i prefer aliased

Reply:
Changed it to aliased.